Testscripts/Linux/xfstesting.sh

```diff
@@ -172,12 +172,21 @@ Main() {
     elif [[ $(detect_linux_distribution) == coreos ]];then
         config_path="/usr/boot/config-$(uname -r)"
     fi
+
     grep -q "CONFIG_BTRFS_FS is not set" $config_path
-    if [ $? -eq 0 ];then
+    if [ $? -eq 0 ] && [ $FSTYP == "btrfs" ];then
         LogMsg "CONFIG_BTRFS_FS is not set in $config_path file in ${DISTRO}, skip the test"
         SetTestStateSkipped
         exit 0
     fi
+
+    grep -q "CONFIG_CIFS is not set" $config_path
```

Comment:
Why not if - else block? You keep them in 2 different conditions. If both fell into False, then what happen?

Reply:
CONFIG_BTRFS_FS is not set and current test file system is btrfs then skip test.
CONFIG_CIFS is not set and current test file system is cifs then skip test.
When both false (not enter into these two condition judgement), it means the test can be run in this system, it could be below situations -
btrfs test and current VM support then run case
cifs test and current VM support then run case
other filesystem like xfs, run case